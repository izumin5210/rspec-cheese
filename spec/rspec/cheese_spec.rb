require "spec_helper"

describe Rspec::Cheese do
  it "has a version number" do
    expect(Rspec::Cheese::VERSION).not_to be nil
  end

  it 'generates snapshot' do
    path = File.join(File.dirname(__FILE__), "__snapshots__", "cheese_spec.rb.yaml.snap")
    snapshots = File.exists?(path) ? YAML.load(File.read(path)) : {}
    expect(snapshots[RSpec.current_example.id]).to say_cheese
  end
end
