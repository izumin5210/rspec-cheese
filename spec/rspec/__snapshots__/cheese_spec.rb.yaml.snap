---
"./spec/rspec/cheese_spec.rb[1:2]": 
